processing/src/main/java/io/druid/query/aggregation/ObjectAggregateCombiner.java

```diff
@@ -26,4 +26,9 @@
  */
 public abstract class ObjectAggregateCombiner<T> implements AggregateCombiner<T>, ObjectColumnSelector<T>
 {
+  @Override
+  public boolean isNull()
```

Comment:
Makes sense to make the method `final` because object selectors must always return false.